core/views.py

```diff
@@ -74,6 +85,22 @@ def get_context_data(self, *args, **kwargs):
         context[self.tab_name] = True
         return context
 
+
+class AccountChangeView(AccountMixin, AccountTabsMixin, PermissionRequiredMixin, UpdateView):
+    template_name = 'core/user_accounts/account_edit.html'
+    form_class = AccountForm
+    success_url = reverse_lazy('core:user_accounts/account')
+    permission_required = ('core.can_view_account_information_self', 'core.can_change_account_information_self')
+    raise_exception = True
+    tab_name = 'tab_account'
+
+
+    def form_valid(self, form):
+        message = _("Your account information has been saved successfully! Notice the name disappears when the user is a membership ")
```

Comment:
The name-hiding for members is basically an ugly hack on our side; there's no need to mention that to our members
```suggestion
        message = _("Your account information has been saved successfully!")
```